index.html

```diff
@@ -724,9 +730,35 @@ <h2>Security Considerations</h2>
 critical systems using the technology outlined in this specification.
     </p>
 
-    <p class="issue">
-Write security considerations.
-    </p>
+    <section class="informative">
+      <h3>Bitstring Encoding</h3>
+
+      <p>
+It is critical that implementers pay particular attention to the way that they
+encode and decode bitstrings. Failure to do so can result in checking the
+wrong bitstring index for a given credential, leading to a misinterpretation
+of its present state (e.g., mistaking a revoked status for an unrevoked
+status). As stated in Section <a href="#statuslist2021credential"></a>,
+bitstrings are encoded such that the first (zeroth) index refers to the
+left-most bit of the bitstring array. The diagram below demonstrates the
+proper layout for an uncompressed bitstring.
+      </p>
+
+      <figure id="bitstring">
+        <img style="margin: auto; display: block; width: 75%;"
+             src="diagrams/bitstring-layout.svg" alt="a diagram showing two
+             wide rectangles shown side-by-side. Each rectangle is partitioned
+             into eight boxes to represent the 8 bits in a byte. The left
+             rectangle is labeled 'First byte' and the right rectangle is
+             labeled 'Last byte'. The left rectangle's left-most bit box is
+             pointed to by an arrow labeled 'index: 0'. The right rectangle's
+             right-most bit box is pointed to by an arrow labeled
+             'index: length - 1'.">
+        <figcaption style="text-align: center;">
+          A visual depiction of the bitstring layout.
+        </figcaption>
+      </figure>
+    </section>
```

Comment:
```suggestion
      </figure>
      
      <p>
For example, if a bitstring is 200,000 bits in size, the first index will be 0, and the last index will be 199,999.
      </p>
    </section>
```

Reply:
@TallTed does this address your concern? I used a number greater than 17,000 because implementers are not supposed to use bitstrings shorter than 131,072 bits (16KB is the minimum allowable herd size).

Reply:
Maybe just use `131,072 bits in size` ... `last index will be `131,071` to keep with the 16KB minimum. Using some size that isn't a power of 2 may lead to more questions. In fact, we might want to explicitly say that all the bit string lengths must be a power of 2 to keep implementations simple -- and that that doesn't mean all bits must be assigned to something, just that the total length will always a power of 2.

Reply:
Updated to use 131,072 bits... feels a bit less understandable than 200,000 vs. 199,999, but meh... not that much less?

Reply:
Works for me.